Upstream-Status: Backport

From fffa60a0693182085a7d4351150137eda26dbf82 Mon Sep 17 00:00:00 2001
From: Guy Harris <guy@alum.mit.edu>
Date: Wed, 13 Jun 2012 12:20:19 -0700
Subject: [PATCH] The leading comma looked weird; remove it.

That might be how the --disable-canusb bug got in.  Hopefully this will
make the code look a little clearer and possibly avoid future bugs of
that sort.
---
 configure.in |    9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/configure.in b/configure.in
index d8554ca..ba5b64e 100644
--- a/configure.in
+++ b/configure.in
@@ -1371,7 +1371,8 @@ AC_SUBST(NETFILTER_SRC)
 
 AC_ARG_ENABLE([bluetooth],
 [AC_HELP_STRING([--enable-bluetooth],[enable Bluetooth support @<:@default=yes, if support available@:>@])],
-,enable_bluetooth=yes)
+    [],
+    [enable_bluetooth=yes])
 
 if test "x$enable_bluetooth" != "xno" ; then
 	dnl check for Bluetooth sniffing support
@@ -1396,7 +1397,8 @@ fi
 
 AC_ARG_ENABLE([canusb],
 [AC_HELP_STRING([--enable-canusb],[enable canusb support @<:@default=yes, if support available@:>@])],
-,enable_canusb=yes)
+    [],
+    [enable_canusb=yes])
 
 if test "x$enable_canusb" != "xno" ; then
 	dnl check for canusb support
@@ -1422,7 +1424,8 @@ fi
 
 AC_ARG_ENABLE([can],
 [AC_HELP_STRING([--enable-can],[enable CAN support @<:@default=yes, if support available@:>@])],
-,enable_can=yes)
+    [],
+    [enable_can=yes])
 
 if test "x$enable_can" != "xno" ; then
 	dnl check for CAN sniffing support
-- 
1.7.9.5

